modules/tts/tts-adapters/tts-adapter-sapinative/.gitignore

```diff
@@ -1,2 +1,10 @@
 build\
-.vs\
\ No newline at end of file
+*.log
+*.vcxproj.*
+src/main/resources/x64/*
+!src/main/resources/x64/sapinative.dll
+!src/main/resources/x64/onecorenative.dll
+src/main/resources/x86/*
+!src/main/resources/x86/sapinative.dll
+!src/main/resources/x86/onecorenative.dll
+build/
```

Comment:
```
*.vcxproj.* -> exclude files generated by visual studio for user preferences
src/main/resources/x64/* -> exclude all generation files that are not the library (can be debug symbols and static libs that could be used for linking in other C++ project)
!src/main/resources/x64/sapinative.dll -> only keep the dll 
!src/main/resources/x64/onecorenative.dll -> only keep the dll
src/main/resources/x86/* -> same as x64 folder
!src/main/resources/x86/sapinative.dll -> same as for x64 folder
!src/main/resources/x86/onecorenative.dll -> same as for x64 folder
build/ -> exclude intermediate build folder of the VS solution
```

Reply:
Thanks. So it's all from Visual Studio. `/packages/` too. What about `*.log` and `./.apt*`?

Reply:
for the log i don't recall exactly but i think it was the error/crash log of the jvm when a memory leak occured on the native side (haven't got one in while since the updates on the memory cleaning in the native code)

For the apt folders, its a visual studio code extension by red hat when working on java project, that adds those folder (i think those are mainly indexes)